src/core_codemods/use_walrus_if.py

```diff
@@ -134,6 +139,11 @@ def leave_If(self, original_node, updated_node):
 
     def leave_Assign(self, original_node: cst.Assign, updated_node: cst.Assign):
         del updated_node
+        if not self.filter_by_path_includes_or_excludes(
+            self.node_position(original_node)
+        ):
+            return original_node
+
```

Comment:
As we discussed before, we only filter the nodes that "anchor" the changes. In this case the `If` node is the one being used for this purpose. Note that the change is reported with the `If` node position. Thus you don't need this check.